src/aioswitcher/device/tools.py

```diff
@@ -211,3 +213,24 @@ def validate_token(username: str, token: str) -> bool:
     else:
         logger.debug("request failed with status code: %s", response.status_code)
     return is_token_valid
+
+
+def get_shutter_index(device_type: DeviceType, device_num: int) -> int:
```

Comment:
This is very weird and requires some explanation in the code docs.

Reply:
Added more docs to the functions description.

Reply:
I think even after the update it is not clear.
Please add a small part that explain the device configuration. something like:
S11 has two lights circuits & one shutter circuits.
Lights circuits are numbered 0 & 1, shutter circuit is 0.

I also think the name `device_num` is confusing as we use device in multiple places here. I don't know what would be the best option but maybe `circuit_number` or `sub_device_number`
I usually try to avoid short names in external params, I prefer it will be clear what it does vs a shorter code.

Reply:
Regarding:
`
S11 has two lights circuits & one shutter circuits.
Lights circuits are numbered 0 & 1, shutter circuit is 0.
`
Where is the best place to write this?

Reply:
@thecode I put this information next to those functions.. I am not sure where to put it if not here..
Also changed to circuit_number

Reply:
From my point it looks clear now